hatchery/alb.go

```diff
@@ -38,13 +38,19 @@ func (creds *CREDS) createTargetGroup(userName string, vpcId string, svc *elbv2.
 			case elbv2.ErrCodeTooManyTagsException:
 				fmt.Println(elbv2.ErrCodeTooManyTagsException, aerr.Error())
 			default:
-				fmt.Println(aerr.Error())
+				Config.Logger.Errorw("Error creating target group",
+					"error", aerr.Error(),
+					"code", aerr.Code(),
+					"message", aerr.Message(),
+					"user", userName,
+				)
 				return nil, err
 			}
 		} else {
-			// Print the error, cast err to awserr.Error to get the Code and
-			// Message from an error.
-			fmt.Println(err.Error())
+			Config.Logger.Errorw("Error creating target group",
+				"error", err.Error(),
+				"user", userName,
```

Comment:
sometimes you use `user` and sometimes `username`, we should use the same field name consistently

Reply:
Good catch! I have changed them all to username 😄